composer.json

```diff
@@ -4,34 +4,34 @@
   "description": "Panther (WebDriver) driver for Mink framework",
   "license": "MIT",
   "require": {
-    "php": "^7.2",
+    "php": "^7.2.5",
     "behat/behat": "^3.6.1",
     "behat/mink": "^1.8.1",
-    "behat/mink-extension": "^2.3.1",
+    "friends-of-behat/mink-extension": "^2.4.0",
     "php-webdriver/webdriver": "^1.8.2",
-    "symfony/browser-kit": "^4.4.8",
-    "symfony/config": "^4.4.8",
-    "symfony/dependency-injection": "^4.4.8",
-    "symfony/dom-crawler": "^4.4.8",
+    "symfony/browser-kit": "^5.0.8",
```

Comment:
I highly recommend you to support Symfony 4.4 too as it's a LTS and still active for years

Reply:
I'm not interested into supporting multiple Symfony major versions.